patterns/portfolio-posts.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Title: List of posts with featured images in three columns
+ * Description: This pattern only shows featured images. Make sure that you have assigned images to your posts.
+ * Slug: twentytwentyfour/portfolio-posts
```

Comment:
I don't care much about the slug but let's rename this file to posts-three-columns so it matches the other 2 query patterns namings